chromium_src/chrome/browser/ui/views/frame/app_menu_button.cc

```diff
@@ -3,8 +3,28 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "chrome/browser/ui/views/frame/app_menu_button.h"
 #include "brave/browser/ui/views/toolbar/brave_app_menu.h"
+#include "ui/views/controls/menu/menu_config.h"
 
+#define AppMenuButton AppMenuButton_ChromiumImpl
 #define AppMenu BraveAppMenu
 #include "src/chrome/browser/ui/views/frame/app_menu_button.cc"
 #undef AppMenu
+#undef AppMenuButton
+
+void AppMenuButton::RunMenu(std::unique_ptr<AppMenuModel> menu_model,
+                            Browser* browser,
+                            int run_flags) {
+  auto& config = const_cast<views::MenuConfig&>(views::MenuConfig::instance());
+  const auto orig_vertical_margin = config.item_vertical_margin;
+  const auto orig_horizontal_padding = config.item_horizontal_padding;
+  config.item_vertical_margin = 9;
+  config.item_horizontal_padding = 16;
+
+  AppMenuButton_ChromiumImpl::RunMenu(std::move(menu_model), browser,
+                                      run_flags);
+
+  config.item_vertical_margin = orig_vertical_margin;
```

Comment:
Does this mean submenus won't get the padding?

Reply:
On Linux, they don't seem to:
![image](https://github.com/brave/brave-core/assets/7678024/3a4ce882-1d63-4be1-854b-1ec5cb297d48)

I think @aguscruiz was thinking of changing it globally, so maybe we can just patch the default `MenuConfig`?

Reply:
Oh, thanks. I missed sub menus.
Yeah, changing it globally would be better. Done. 👍🏼

Reply:
Was about to come here and comment just that, yeah it should be global 🙏